geanypy/src/Makefile.am

```diff
@@ -5,11 +5,11 @@ geanyplugin_LTLIBRARIES		=	geanypy.la
 geanyplugindir				=	$(libdir)/geany
 
 geanypy_la_LDFLAGS			=	-module -avoid-version -Wl,--export-dynamic
-geanypy_la_CPPFLAGS			=	@GEANY_CFLAGS@ @PYGTK_CFLAGS@ @PYTHON_CPPFLAGS@ \
+geanypy_la_CPPFLAGS			=	@PYTHON_CPPFLAGS@ \
 								-DGEANYPY_PYTHON_DIR="\"$(libdir)/geany/geanypy\"" \
 								-DGEANYPY_PLUGIN_DIR="\"$(libdir)/geany\"" \
 								-DG_LOG_DOMAIN=\"GeanyPy\"
-geanypy_la_CFLAGS			=	@GEANYPY_CFLAGS@ @GMODULE_CFLAGS@
+geanypy_la_CFLAGS			=	@PYGTK_CFLAGS@ @GEANY_CFLAGS@ @GEANYPY_CFLAGS@ @GMODULE_CFLAGS@
```

Comment:
I don't think this changes matters much, but IMO it's not bad, rather good.  Yes, *pkg-config* will mostly fill the `*_CFLAGS` variable with preprocessor flags, but it might put other stuff here if it wants.  Also, it's fine putting preprocessor flags in `*_CFLAGS` AFAIK, only library order matters.